bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/TimeZoneProvider.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.i18n;
+
+import java.util.TimeZone;
+
+/**
+ * This interface describes a provider for a location and time zone.
+ *
+ * @author Erdoan Hadzhiyusein - Initial contribution and API
+ */
+public interface TimeZoneProvider {
+
+    /**
+     * Provides access to the time zone property.
+     *
+     * @return the time zone set in the Paper UI and if there isn't one, the default time zone of the underlying system
+     */
+    TimeZone getTimeZone();
```

Comment:
If I understood \@maggu2810's question right, he was asking wether it might make more sense to return a `java.time.ZoneId` here instead of a `java.util.TimeZone`. Considering that with the `java.time.*` classes that you are using in #4259 you always have to call `timeZone.toZoneId()` on it anyway, IMHO this would make sense. Or did I miss anything?